pkg/apis/operator/v1alpha1/types.go

```diff
@@ -526,6 +529,82 @@ type GardenerSchedulerConfig struct {
 	LogLevel *string `json:"logLevel,omitempty"`
 }
 
+// GardenerDashboardConfig contains configuration settings for the gardener-dashboard.
+type GardenerDashboardConfig struct {
+	// EnableTokenLogin specifies whether it is possible to log into the dashboard with a JWT token. Either this or OIDC
+	// must be configured.
+	// +kubebuilder:default=true
+	// +optional
+	EnableTokenLogin *bool `json:"enableTokenLogin,omitempty"`
+	// FrontendConfigMapRef is the reference to a ConfigMap containing the frontend configuration.
+	// +optional
+	FrontendConfigMapRef *corev1.LocalObjectReference `json:"frontendConfigMapRef,omitempty"`
+	// GitHub contains configuration for the GitHub ticketing feature.
+	// +optional
+	GitHub *DashboardGitHub `json:"gitHub,omitempty"`
```

Comment:
I don't know whether the Dashboard is using this for issues only, or whether it will add another feature with GitHub tomorrow. For now, I chose to use `gitHub` in the `Garden` API because that's also how it's called in the Dashboard config itself.